docs/stdlib/enum.rst

```diff
@@ -11,24 +11,24 @@ Enums
 
     :index: enum
 
-    An enumerated type is a data type consisting of an ordered list of values.
+    Represents an enumeration of values within an ordered list.
 
-    An enumerated type can be declared in a schema declaration using
-    the following :ref:`syntax <ref_datamodel_enums>`:
+    An enumerator type can be declared in a schema definition by using
```

Comment:
I think "enumerator" is not correct here. `Enum` is short for "enumeration" or "enumerated type". The term "enumerator" makes me think of something that is iterable/loopable, in that you can "enumerate" all the values. https://en.wikipedia.org/wiki/Enumerated_type 

I think the clearest and most explicit thing it just to refer to these as `"enum types"` - that term is more widespread than "enumerated types" these days.

Reply:
Agree with @colinhacks here. Just wanted to log that we should make this change global by updating other mentions of this type, like those on lines 23 and 31.